### DISCUSSION TITLE
Default "system" message

### DISCUSSION THREAD

**Discussion by martin-hoeger** [intellibar/main#46] [General]
Is there a way to set a default "system" message for all the chats? Something like:

> Use professional technical vocabulary. Use simple language and simple sentences. I need a short answer that assumes a high level of familiarity with the subject.

**Comment by erusev** [comment]
Hi, this is coming soon! Meanwhile, as a workaround, you can create a command in Settings with the instruction you want and use that when starting a new chat.
